drivers/encx24j600/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = encx24j600
```

Comment:
if I learned it correctly, you can remove this line as the module name will be automatically set